pkg/controller/infinispan/infinispan_controller.go

```diff
@@ -95,8 +95,11 @@ func secondaryResourceTypes() []SecondaryResourceType {
 			}},
 		},
 		{&corev1.Secret{}, predicate.Funcs{
+			UpdateFunc: func(e event.UpdateEvent) bool {
+				return true
+			},
 			DeleteFunc: func(e event.DeleteEvent) bool {
-				return false
+				return true
```

Comment:
@ryanemerson why you disable events from the `Secret` delete? What happens if someone delete Secret manually?